dmorg.tex

```diff
@@ -150,6 +150,32 @@ \subsubsection{Hardware Configuration Control}
 
 The sizing model for compute hardware purchasing is detailed in \citeds{LDM-144}, \citeds{LDM-141}, and \citeds{LDM-138}.
 
+\subsection {Release Management } \label{sect:release}
+
+
+The DMCCB maintains  the release plan, \citeds{LDM-564},  synchronized with  the project milestones.
+
+All releases will be identified by a release issue - but we wish to move to Jira release functionality.
```

Comment:
I'm not sure I understand what this means, and I'm not sure we agree on what I think it might means. Isn't this something that the DM-CCB should discuss before we baseline it?

Reply:
We can let this point for future discussions, and remove the second part of the sentence.